docs/structured-streaming-kafka-integration.md

```diff
@@ -703,11 +703,99 @@ Kafka broker configuration):
 
 After obtaining delegation token successfully, Spark distributes it across nodes and renews it accordingly.
 Delegation token uses `SCRAM` login module for authentication and because of that the appropriate
-`spark.kafka.sasl.token.mechanism` (default: `SCRAM-SHA-512`) has to be configured. Also, this parameter
+`spark.kafka.clusters.${cluster}.sasl.token.mechanism` (default: `SCRAM-SHA-512`) has to be configured. Also, this parameter
 must match with Kafka broker configuration.
 
 When delegation token is available on an executor it can be overridden with JAAS login configuration.
 
+#### Configuration
+
+Delegation tokens can be obtained from multiple clusters and <code>${cluster}</code> is an arbitrary unique identifier which helps to group different configurations.
+
+<table class="table">
+<tr><th>Property Name</th><th>Default</th><th>Meaning</th></tr>
+  <tr>
+    <td><code>spark.kafka.clusters.${cluster}.bootstrap.servers</code></td>
```

Comment:
What do you think of changing these names?

I think "bootstrap.servers" and "target.bootstrap.servers" ends up being a little confusing. Maybe call this "auth.servers" to make it more explicit that these are only used for the initial authentication.

Reply:
Good idea, such way not only the doc describes the intention. Changed.

Reply:
As I'm thinking about this maybe it would be better to call it `auth.bootstrap.servers` since `bootstrap.servers` in Kafka has a defined format and can be a subset of all brokers. This way the name would tell both (only auth + format). What do you think?

Reply:
Leaving this open in case you disagree.